pkg/reconciler/vsphere/controller.go

```diff
@@ -69,10 +74,27 @@ func NewController(
 		Handler:    controller.HandleAll(impl.EnqueueControllerOf),
 	})
 
+	rbacInformer.Informer().AddEventHandler(cache.FilteringResourceEventHandler{
+		FilterFunc: controller.FilterGroupKind(v1alpha1.Kind("VSphereSource")),
+		Handler:    controller.HandleAll(impl.EnqueueControllerOf),
+	})
+
+	// Don't trigger off of CM updates because we don't care about the content
```

Comment:
That's why I didn't add it, curious why add the commented out code here? Comment is fine, but curious why add teh code for it.

Reply:
I noticed it while fixing merge conflicts and added both then thought better of it.  I'll clean it up with the other stuff in a sec.